package.json

```diff
@@ -10,7 +10,7 @@
   },
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.0.0-beta.50",
+    "@apollo/client": "^3.2.5",
```

Comment:
We should probably upgrade it anyways, this combination of packages and versions is what we've been developing RTU with anyways so if possible it would be preferable for us to bump this version

Reply:
`apollo-client` is actually a dependency of `offix-client` because offix essentially builds on top of `apollo-client`, adding the additional functionality of offline, etc.

Reply:
let's address this later then